### ISSUE TITLE
Does the PTL training provide the same accuracy as the PT training?

### ISSUE THREAD

**Issue by mohaghighat** [nSightSurgical/yolov7#3]
After https://github.com/nSightSurgical/yolov7/issues/1 is completed, train a PyTorch Lightning model with the same data and the hyperparameters as the [20220927](https://docs.google.com/spreadsheets/d/1j7eHeYDi69tFtQRtuUL95frO9STY-mQgRcXe7rFiGOY/edit#gid=231011500&range=12:12) model, and verify the accuracy of the model is the same as before.
